modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/adapter/SW360ReleaseClientAdapter.java

```diff
@@ -150,4 +153,18 @@ public SW360Release createRelease(SW360Release releaseFromArtifact, boolean uplo
         }
         return Optional.empty();
     }
+
+    public List<SW360SparseAttachment> getAttachmentsOfRelease(SW360Release release, HttpHeaders header) {
+        return releaseClient.getItemAttachments(release, header);
+    }
+
+    public Optional<Path> downloadAttachment(SW360Release release, SW360SparseAttachment attachment, Path downloadPath, HttpHeaders header) {
+        Self self = release.get_Links().getSelf();
```

Comment:
Nicer would be
`return Optional.of(release).map(...get_Links()).map(...getSelf()).map(...getHref()).map(...releaseClient...);`